libraries.json

```diff
@@ -0,0 +1,16 @@
+{
+	"Angular": {
+		"imports": ["angular"],
```

Comment:
I'd change this and deal with AngularJS later

```suggestion
		"imports": ["@angular/"],
```

Reply:
Will do this in the next PR